webpack/assets/javascripts/foreman_tools.js

```diff
@@ -44,6 +44,11 @@ export function activateTooltips(el = 'body') {
                                    }
                               });
   el.find('*[title]').not('*[rel]').tooltip({ container: 'body' });
+
+  // adding this to remove displayed tooltips for react-initiated
+  // calls to this function (no page:restore fired).
+  $('.tooltip.in').remove();
```

Comment:
why is this here? shouldnt we avoid the in css class in the inigital tooltip initalization? e.g. https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/foreman_tools.js#L40